planner/type_join.go

```diff
@@ -444,9 +444,9 @@ func fetchPrimaryDoc(node, subNode planNode, parentProp string) (bool, error) {
 	if scan == nil {
 		return false, nil
 	}
-	rootDocKey := base.MakeDocKey(scan.col.Description(), docKeyStr)
+	dsKey := base.MakeDSKeyWithCollectionAndDocID(scan.col.Description(), docIDStr)
```

Comment:
todo: Same as my comment on planner/commit.go - please do not call this `dsKey`

Reply:
Will rename.

- [ ] rename: `dsKey` -> `docID`

Reply:
I also oppose this todo. Sorry Andy.